client/reader/update-notice/_style.scss

```diff
@@ -16,8 +15,12 @@
 	pointer-events: none;
 	transition: background 0.15s ease-in-out, transform 0.20s cubic-bezier( 0.175, 0.885, 0.320, 1.275 );
 
-	@include breakpoint( ">660px" ) {
-		top: ( 47px + 8px ); // Make way for the sticky Masterbar
+	@include breakpoint( ">480px" ) {
+		right: 16px;
+	}
+
+	@include breakpoint( "<480px" ) {
+		right: 8px;
```

Comment:
I think this media query breakpoint can be avoided by just changing line [6](https://github.com/Automattic/wp-calypso/pull/1160/files#diff-8310ef573196d355de1db6362ac82a15L6) above to `right: 8px;`